.github/workflows/simple-task-test.yml

```diff
@@ -3,19 +3,28 @@
 name: Simple Task "simple-task-test"
 
 on:
+  pull_request:
+    types: [opened, synchronize]
   push:
     branches:
     tags:
   workflow_dispatch:
+    inputs:
+      delay_seconds:
+        description: 'Time to delay'
+        required: false
+        default: 60
 
 jobs:
   simple-task-test:
     runs-on: ubuntu-latest
 
     steps:
       - name: delay execution if manually executed
-        run: sleep 1m
-        if: ${{ github.event == 'workflow_dispatch' }}
+        env:
```

Comment:
I think we need to delay this anyway, I see no reason for this test workflow to run _fast_. It should run for at least 60 seconds (or even 90) to give time for dependent checks to initialize and _wait_